hack/package.txt

```diff
@@ -0,0 +1,15 @@
+gardener/gardener/extensions/
```

Comment:
Inline this file in the script. `hack/package.txt` is a very generic file name and might confuse folks where it is used. Inlining it in the script where it is used, is much clearer.